### PR TITLE
[SPARK-47081][CONNECT][FOLLOW-UP] Add the `shell` module into PyPI package

### PR DESCRIPTION
**Pull request by HyukjinKwon**
### What changes were proposed in this pull request?

This PR is a followup of https://github.com/apache/spark/pull/45150 that adds the new `shell` module into PyPI package.

### Why are the changes needed?

So PyPI package contains `shell` module.

### Does this PR introduce _any_ user-facing change?

No, the main change has not been released yet.

### How was this patch tested?

The test case will be added at https://github.com/apache/spark/pull/45870. It was found out during working on that PR.

### Was this patch authored or co-authored using generative AI tooling?

No.

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
cc @grundprinzip @zhengruifeng

File: python/packaging/classic/setup.py [hunk 1]
```diff
@@ -277,6 +277,7 @@ def run(self):
             "pyspark.sql.connect.proto",
             "pyspark.sql.connect.streaming",
             "pyspark.sql.connect.streaming.worker",
+            "pyspark.sql.connect.shell",
```
Comment (HyukjinKwon) [inline python/packaging/classic/setup.py:280]:
```suggestion
            "pyspark.sql.connect.proto",
            "pyspark.sql.connect.shell",
            "pyspark.sql.connect.streaming",
            "pyspark.sql.connect.streaming.worker",
```

**Review by xinrong-meng — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master.

**Review by dongjoon-hyun — COMMENTED** [review]
+1, LGTM.

**Pull request closed without merging**